clearwater-diags-monitor/usr/share/clearwater/bin/gather_diags_and_report_location

```diff
@@ -13,7 +13,10 @@ DIAGS_LOCATION=/var/clearwater-diags-monitor/dumps/
 
 # In some scenarios we are run as root, and $HOME is not defined.
 # Instead, we should create a dump in the Clearwater user's directory.
-if [ ! -z $HOME ]
+if [ ! -z $SUDO_USER ]
+then
+  eval CLEARWATER_USER_HOME="~$SUDO_USER"
```

Comment:
Add a comment explaining what this command does - it will take users a little while to figure it out otherwise.